### ISSUE TITLE
RW Scaffold results in linter issue with generated Service file

### ISSUE THREAD

**Issue by justinkuntz** [redwoodjs/redwood#5738]
Quick Overview:

After upgrading to RW v2.0.0, running the `yarn rw lint --fix` cleans up the Services files swapping `import { db } from 'src/lib/db'` to be below:
`import type {
QueryResolvers,
  MutationResolvers,
  UserResolvers,
} from 'types/graphql'`

Problem:
Running `yarn redwood generate scaffold <model>` generates Pages, SDL, and Services files based on a given DB schema Model. However, the resulting Services file places the `import { db } from 'src/lib/db'` at the top of the file causing mr. linter to create crazy red squiggles. 

<img width="459" alt="image" src="https://user-images.githubusercontent.com/940266/173170819-c86ee4c5-88a0-4a19-a931-fe5db3012662.png">

Opening the file and saving it corrects the issue.

Expected:
Running `yarn redwood generate scaffold <model>` generates Pages, SDL, and Services files based on a given DB schema Model with the correct order of imports that Redwood enforces in v2.0.0.
<img width="459" alt="image" src="https://user-images.githubusercontent.com/940266/173170846-44e02ee6-bbe1-4f7b-b6a0-2e062018f843.png">

**Comment by jtoar** [comment]
Thanks for the report @justinkuntz, I was able to confirm this locally. I saw red squiggles not only in the service files, but also in a host of others on the web side as well. Did you also see errors in files on the web side?

For the service files, linting the templates makes sense. But the other files were existing code that was modified, so for those it seems like the only solution is to run lint or prettier (via lint) afterwards.
